src/services/activityReports.js

```diff
@@ -345,17 +333,11 @@ export async function populateRecipientInfo(activityRecipients, grantPrograms) {
       );
 
       // Name.
-      let nameValue;
-      if (recipient.grant.recipient) {
-        nameValue = `${recipient.grant.recipient.name} - ${recipient.grant.numberWithProgramTypes}`;
-      } else {
-        nameValue = `${recipient.grant.numberWithProgramTypes}`;
-      }
       Object.defineProperty(
         recipient.grant,
         'name',
         {
-          value: nameValue,
+          value: recipient.name,
```

Comment:
Are you sure line 340 and 349 should be the same value? one is the recipient the other is the recipient.grant. It would make more sense if the recipient.grant was the grant number.